src/forms/Household.js

```diff
@@ -219,12 +220,13 @@ const MemberField = function ({ household, time, setHousehold, setClientProperty
       </Columns.Two>
 
       <Columns.Three>
-        <Input
-          className = {time + '-member-age ' + time}
-          onChange  = {onMemberChange}
+        <ManagedNumberField
           value     = {member.m_age}
           name      = {'m_age'}
-          type      = {'number'} step = {'1'} min = {'0'} />
+          className = {time + ' member-age ' + time}
+          validate  = {isPositiveWholeNumber}
+          format    = {function ( value ) { return value; }}
```

Comment:
The formatter will accept `023`. Super minor.

Reply:
I'm ok with it for now unless someone else objects to it. It's good to keep in mind, though, and maybe we'll deal with it when beta testing is going on.